ReactiveCocoa/UIKit/UIControl.swift

```diff
@@ -44,11 +44,23 @@ extension Reactive where Base: UIControl {
 	/// - parameters:
 	///   - controlEvents: The control event mask.
 	///
-	/// - returns: A signal that sends the control each time the control event 
-	///            occurs.
+	/// - returns: A signal that sends the control each time the control event occurs.
 	public func controlEvents(_ controlEvents: UIControlEvents) -> Signal<Base, NoError> {
+		return mapControlEvents(controlEvents, { $0 })
+	}
+
+	/// Create a signal which sends a `value` event for each of the specified
+	/// control events.
+	///
```

Comment:
Can we add a `- note` or `- important` here that explains why this is needed? (i.e. that events can be sent recursively, so you may want to transform if…)